### ISSUE TITLE
Accessibility Issue entering pedal marks with keyboard/screen reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#19522]
### Issue type

UI bug

### Bug description

There are a number of problems around entering pedal marks with keyboard and NVDA.
The screen reader does not announce anything or return focus to the score after the palette selection most of the time, so it is easy to inadvertently add twice not realising it is done.
The place to adjust or delete the pedal mark says "dummy panel pedal" but not what type e.g. both hooks angled" and would be much more efficient if the word pedal was first.
When you use Shift + right arrow to extend the line the screen reader announces nothing.
Copy & Paste does not work - I think it is putting in the pedal mark but seems to lose the line length.  It says start of pedal on the note it is pasted to but I can't find an end.

### Steps to reproduce

Select a note and using the keyboard tab to and open the keyboard palette.  Arrow down to both ends hooked and press enter.  Most times nothing is said and the focus does not return to the score.  press Esc.  Assuming it has entered the pedal mark use Alt + right arrow to move on to the pedal adjustment thingy, you will hear "dummy panel pedal".  Use Shift plus right arrow to extend the pedal line.  You will not hear any announcement of where it is being moved to.
Press Alt left arrow to move back to the dummy panel pedal and press Ctrl C to copy, move to a new note and press Ctrl V to paste.  Now when you Alt arrow onto this new panel (or where it should be) the screen reader says nothing.  Arrow back to the note you pasted onto and you will hear start of pedal after the note announcement, arrow along the notes, there does not seem to be an end of pedal anywhere.
Expected Behaviour: 
to return to the score and announce something so the user knows the pedal mark has been inserted.
To speak the type of pedal mark.
To speak beats/bar as the line is extended.
To be able to copy and paste multiple times as currently inserting 1 pedal mark takes a minimum of 18 key presses plus additional to go further down the palette and to extend the line.  Keyboard shortcuts for these would also be welcome.

### MuseScore Version

4

### Regression

I don't know

### Operating system

Windows 10

**Comment by jmwmusic** [comment]
Great to see that in 4.2 I now seem to be able to copy & paste pedal marks.  Weirdly though if I select a range and add a pedal mark when I go back and Alt arrow over it it is speaking nothing with NVDA, not even the old dummy panel.  If I have just a single note selected it now reads as pedal when Alt arrowing over it.  Just wondering if this is a clue to other range selection issues.

**Comment by jmwmusic** [comment]
In case it helps I have identified that i get the exact same result with entering hairpin and 8va from the palette.  however, when adding these from the Add menu they read correctly after insertion with a selected range.  Note this is without crossing the Braille panel and without a slur in the range.

**Comment by jmwmusic** [comment]
I am closing this issue as the copy and paste is now fixed.  The other issues affect a number of other lines also and so are covered in#19932,  #28039, #19525, #28038 & #28037.
